.github/workflows/ci.yml

```diff
@@ -0,0 +1,226 @@
+---
+name: "CI"
+on:  # yamllint disable
+  - "push"
+  - "pull_request"
+
+jobs:
+  black:
+    runs-on: "ubuntu-20.04"
+    env:
+      INVOKE_NAUTOBOT_LOCAL: "True"
+    steps:
+      - name: "Check out repository code"
+        uses: "actions/checkout@v2"
+      - name: "Setup environment"
+        uses: "networktocode/gh-action-setup-poetry-environment@v1"
+      - name: "Linting: black"
+        run: "poetry run invoke black"
```

Comment:
what version of flake/black will this end up using, is it from Nautobot's pyproject.toml or form the networktocode/gh-action-setup-poetry-environment@v1?

Reply:
If I'm reading it right, https://github.com/networktocode/gh-action-setup-poetry-environment/blob/main/action.yml#L32 runs `poetry install` so it should bring in the version that's defined in Nautobot's pyproject.toml.

Reply:
@nniehoff, as @glennmatthews mentioned, it installs `black` from the `pyproject.toml`.